cromwell-helm/templates/reverse-proxy-config.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "app.fullname" . }}-proxy-config
+data:
+  {{ .Values.config.proxy.conf_file }}: |-
+    daemon off;
+
+    events {
+      worker_connections 1024;
+    }
+    http {
+      server {
+        listen 8000;
+
+        # For now, keep serving 'cromwell' at proxy root, but we should consider this deprecated and
+        # update clients to use `/cromwell/` when talking to the Cromwell service.
+        location / {
```

Comment:
So for now, when we load the page that we get from `proxyUrl` through Leo, it will open the Cromwell Swagger page and not the nginx page described below?

Reply:
Correct, the nginx page is hosted on `/index.html`, if you know to look for it

Reply:
Ah I see 👍